src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -114,25 +113,26 @@ ID.me. To go through the ID.me identity-proofing process, you’ll need:</p>
             <li className="process-step list-four">
               <div><h5>VA Review</h5></div>
               <p>We process applications in the order we receive them.</p>
-              <p>We’ll review your application to verify your eligibility. We may contact you if we have any questions or need more information.</p>
+              <p>We’ll review your application to check your eligibility. We may contact you if we have any questions or need more information.</p>
             </li>
             <li className="process-step list-five">
               <div><h5>Decision</h5></div>
-              <p>Once we verify your application, you should receive your Veteran ID Card by mail in about 60 days.</p>
+              <p>Once we approve your application, you should receive your Veteran ID Card by mail in about 60 days.</p>
             </li>
           </ol>
         </div>
+        <h2 className="help-heading">Ready to get started?</h2>
```

Comment:
Goal was to have this look like the Need help? heading- want to make sure it's okay to reuse the styling and h2 or if I should duplicate it into the Intro page css.